util/envDynawo.sh

```diff
@@ -1049,6 +1052,26 @@ build_tests_coverage() {
   fi
 }
 
+clean_models() {
+  for model in `echo $@`; do
```

Comment:
Is this OK if there is not argument? (just to make sure it would not erase the P folder)

Reply:
It was ok nothing happened but I added a test to give the information to the user